packages/medusa/src/api/routes/store/payment-collections/refresh-payment-session.ts

```diff
@@ -10,21 +10,6 @@ import { PaymentCollectionService } from "../../../../services"
  * parameters:
  *   - (path) id=* {string} The id of the PaymentCollection.
  *   - (path) session_id=* {string} The id of the Payment Session to be refreshed.
- * requestBody:
- *   content:
- *     application/json:
- *       schema:
- *         type: object
- *         required:
- *           - provider_id
- *           - customer_id
- *         properties:
- *           provider_id:
- *             description: The Payment Provider id.
- *             type: string
- *           customer_id:
- *             description: The Customer id.
- *             type: string
```

Comment:
Removed requestBody declaration. Endpoint only expects path parameters.

Reply:
I know this is out of the scope of this PR, but after seeing this comment I can also see that in the JS Client code example a `payload` is passed as a third parameter to `refreshPaymentSession`, however looking at this method it actually doesn't accept a payload. We should remove it from the example.

Reply:
Fixed in 57840eec970ff0811d63bf6d8f991587fd14dd48